### DISCUSSION TITLE
FORCE set screen resolution

### DISCUSSION THREAD

**Discussion by YuvanMichaelVivenzi** [quickemu-project/quickemu#1499] [Quickemu] [answered]
I have installed macOS mojave and I got to the welcome screen. The mouse grab cuts off at exactly half of the screen. Resolution I have on host is 1366x768. Using 1360x768 for vm, which unfortunately gets reset when apple logo comes, back to qemu's default when I want 1360x768 it resets to the default qemu one.

**Comment by YuvanMichaelVivenzi** [comment, marked as answer]
macOS catalina works, Makes sense to remove macOS mojave as none of the features of quickemu work for mojave but they do for catalina.

**Reply by M-M5** [reply]
Did you do anything different to catalina? I'm currently using catalina but I'm having the exact issue as you now.
